src/Nethermind/Nethermind.Db.Rocks/DbOnTheRocks.cs

```diff
@@ -183,6 +186,11 @@ private RocksDb Open(string path, (DbOptions Options, ColumnFamilies? Families)
                 }
             }
 
+            if (_perTableDbConfig.EnableFileWarmer)
```

Comment:
Minor - Can we remove this general `EnableFileWarmer` flag and already use the one related to actual db? It would be better for user to set just 1 flag instead of 2.

Reply:
The general one is to set the default for other db. Only the state flag need to be specified.